src/store/plugins/fungibleTokens.js

```diff
@@ -158,6 +161,57 @@ export default (store) => {
           decodedResult !== undefined ? 'change_allowance' : 'create_allowance'
         ](activeNetwork.tipContractV2.replace('ct_', 'ak_'), allowanceAmount);
       },
+      async getContractTokenPairs(
+        { rootState: { sdk }, state: { availableTokens }, rootGetters: { account } },
+        contractAddress,
+      ) {
+        try {
+          const tokenContract = await sdk.getContractInstance({
+            source: pairInterface,
+            contractAddress,
+          });
+
+          const [
+            { decodedResult: balances },
+            { decodedResult: balance },
+            { decodedResult: token0 },
+            { decodedResult: token1 },
+            { decodedResult: reserves },
+            { decodedResult: totalSupply },
+          ] = await Promise.all([
```

Comment:
If any of them will fail, we would end up without any info.

Reply:
without all pieces of information, we can't calculate the Pooled tokens amount
if one fails the other information will be useless